deploy/helm/magda-core/charts/opa/templates/deployment.yaml

```diff
@@ -17,7 +17,7 @@ spec:
 {{- end }}
       containers:
       - name: opa
-        image: openpolicyagent/opa:0.10.7
+        image: openpolicyagent/opa:0.17.1
```

Comment:
We probably need to update deploy/helm/magda-core/charts/opa/docker-compose.yml as well

Reply:
👍